src/policy/feerate.cpp

```diff
@@ -37,6 +37,26 @@ CAmount CFeeRate::GetFee(size_t nBytes_) const
     return nFee;
 }
 
+CAmount CFeeRate::GetFeeV1(size_t nBytes_) const
+{
```

Comment:
it's cleaner to use an optional parameter isProtocolV1 for the GetFee() instead of cloning the function